gtk/src/default/gtk-3.0/_colors.scss

```diff
@@ -88,3 +88,8 @@ $coloured_focus_border_color: transparentize(white, 0.3);
 // Yaru: Ignore transformations when using dark accent colors
 $progress_bg_color: $accent_bg_color;
 $checkradio_bg_color: $accent_bg_color;
+
+// Optimize accent-color definition for default background colors
+@import 'sass-utils';
+$yaru-accent-color: optimize-contrast($bg_color, $accent_bg_color);
```

Comment:
I think this var should be named $link-color because actually this one isn't used at all :D

Reply:
defintely, bad copy and paste...